### PR TITLE
Add Dark Sky documentation

### PR DESCRIPTION
**Pull request by fabaff**
With the name change the `forecast` platform will be deprecated and replaced by the `dark_sky` sensor platform.

### CODE REVIEW

File: source/_components/sensor.dark_sky.markdown [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+footer: true
+logo: dark_sky.png
+ha_category: Weather
+featured: true
+ha_release: 0.30
+---
+
+The `dark_sky` platform uses the [Dark Sky](https://darksky.net/) web service as a source for meteorological data for your location. The location is based on the `longitude` and `latitude` coordinates configured in your `configuration.yaml` file. The coordinates are auto-detected but to take advantage of the hyper-local weather reported by Dark Sky, you can refine them down to your exact home address. GPS coordinates can be found by using [Google Maps](https://www.google.com/maps) and clicking on your home or [Openstreetmap](http://www.openstreetmap.org/).
+
+You need an API key which is free but requires [registration](https://developer.forecast.io/register). You can make up to 1000 calls per day for free which means that you could make one approximately every 86 seconds.
```
Comment (Landrash) [inline source/_components/sensor.dark_sky.markdown]: Link should probaly be https://darksky.net/dev/register

File: source/_components/sensor.dark_sky.markdown [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    - temperature_max
+    - temperature_min
+    - apparent_temperature_max
+    - apparent_temperature_min
+    - precip_intensity_max
+```
+
+Configuration variables:
+
+- **api_key** (*Required*): Your API key for http://forecast.io/.
```
Comment (Landrash) [inline source/_components/sensor.dark_sky.markdown]: Link to https://darksky.net/

File: source/_components/sensor.dark_sky.markdown (hunk 1 above)
Comment (fabaff) [inline source/_components/sensor.dark_sky.markdown]: Thanks

File: source/_components/sensor.dark_sky.markdown (hunk 2 above)
Comment (fabaff) [inline source/_components/sensor.dark_sky.markdown]: Thanks

File: source/_components/sensor.forecast.markdown [hunk 3]
```diff
@@ -9,10 +9,14 @@ sharing: true
 footer: true
 logo: forecast.png
 ha_category: Weather
```
Comment (fabaff) [inline source/_components/sensor.forecast.markdown]: Good point.

**Comment by Landrash** [comment]
Looks good! :dolphin:

**Comment by fabaff** [comment]
I guess that we need to merge in the changes from #990 as those are going to the Forecast page.

**Comment by fabaff** [comment]
I guess that we need to merge in the changes from #990 as those are going to the Forecast page.

**Comment by Landrash** [comment]
This is the bigger change and should have priority. 
#990 is a small addition and can be added afterwards. I can do that after this one is merged.
The Darksky change will wait until 0.30 or will you push it into 0.29?

**Comment by fabaff** [comment]
We are waiting for the dependency and Paulus released 0.29 a couple of hours ago. So, it will be in 0.30 (i guess) but not sure if there is enough time to make it available as Weather component ;-)

**Comment by robbiet480** [comment]
@fabaff @Landrash what's the status on this / #990? Good to merge?

**Comment by fabaff** [comment]
First we need a new python-forecastio release.

**Pull request merged**